basicswap/static/js/modules/config-manager.js

```diff
@@ -58,86 +51,23 @@ const ConfigManager = (function() {
-            { symbol: 'LTC', name: 'litecoin', usesCryptoCompare: true, usesCoinGecko: true, historicalDays: 30 },
-            { symbol: 'DOGE', name: 'dogecoin', usesCryptoCompare: true, usesCoinGecko: true, historicalDays: 30 },
-            { symbol: 'DCR', name: 'decred', usesCryptoCompare: true, usesCoinGecko: true, historicalDays: 30 },
-            { symbol: 'NMC', name: 'namecoin', usesCryptoCompare: true, usesCoinGecko: true, historicalDays: 30 },
-            { symbol: 'WOW', name: 'wownero', usesCryptoCompare: false, usesCoinGecko: true, historicalDays: 30 }
-        ],
-
-        coinMappings: {
-            nameToSymbol: {
-                'Bitcoin': 'BTC',
-                'Litecoin': 'LTC',
-                'Monero': 'XMR',
-                'Particl': 'PART',
-                'Particl Blind': 'PART',
-                'Particl Anon': 'PART',
-                'PIVX': 'PIVX',
-                'Firo': 'FIRO',
-                'Zcoin': 'FIRO',
-                'Dash': 'DASH',
-                'Decred': 'DCR',
-                'Namecoin': 'NMC',
-                'Wownero': 'WOW',
-                'Bitcoin Cash': 'BCH',
-                'Dogecoin': 'DOGE'
-            },
-
-            nameToDisplayName: {
-                'Bitcoin': 'Bitcoin',
-                'Litecoin': 'Litecoin',
-                'Monero': 'Monero',
-                'Particl': 'Particl',
-                'Particl Blind': 'Particl Blind',
-                'Particl Anon': 'Particl Anon',
-                'PIVX': 'PIVX',
-                'Firo': 'Firo',
-                'Zcoin': 'Firo',
-                'Dash': 'Dash',
-                'Decred': 'Decred',
-                'Namecoin': 'Namecoin',
-                'Wownero': 'Wownero',
-                'Bitcoin Cash': 'Bitcoin Cash',
-                'Dogecoin': 'Dogecoin'
-            },
-
-            idToName: {
-                1: 'particl', 2: 'bitcoin', 3: 'litecoin', 4: 'decred', 5: 'namecoin',
-                6: 'monero', 7: 'particl blind', 8: 'particl anon',
-                9: 'wownero', 11: 'pivx', 13: 'firo', 17: 'bitcoincash',
-                18: 'dogecoin'
-            },
-
-            nameToCoinGecko: {
-                'bitcoin': 'bitcoin',
-                'monero': 'monero',
-                'particl': 'particl',
-                'bitcoin cash': 'bitcoin-cash',
-                'bitcoincash': 'bitcoin-cash',
-                'pivx': 'pivx',
-                'firo': 'firo',
-                'zcoin': 'firo',
-                'dash': 'dash',
-                'litecoin': 'litecoin',
-                'dogecoin': 'dogecoin',
-                'decred': 'decred',
-                'namecoin': 'namecoin',
-                'wownero': 'wownero'
-            }
+        get coins() {
+            return window.CoinManager ? window.CoinManager.getAllCoins() : [
+                { symbol: 'BTC', name: 'bitcoin', usesCryptoCompare: false, usesCoinGecko: true, historicalDays: 30 },
```

Comment:
not sure if false should be true
```suggestion
                { symbol: 'BTC', name: 'bitcoin', usesCryptoCompare: true, usesCoinGecko: true, historicalDays: 30 },
```

could use coin-manager js here?

Reply:
Everything is using CG now - besides the graphs.